packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -7,20 +7,32 @@ import {createRemoteComponent} from '@remote-ui/core';
 type CustomerStandardMetafieldDependency = 'facts.birth_date';
 
 export interface CustomerSegmentTemplateProps {
-  /* Localized title of the template. */
+  /**
+   * Localized title of the template.
```

Comment:
All of these descriptions should start with an article - "The localized title ..."